sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/SymptomsTabSteps.java

```diff
@@ -0,0 +1,273 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.steps.web.application.cases;
+
+import static org.sormas.e2etests.pages.application.cases.SymptomsTabPage.*;
+
+import com.google.common.truth.Truth;
+import cucumber.api.java8.En;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import javax.inject.Inject;
+import javax.inject.Named;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pages.application.NavBarPage;
+import org.sormas.e2etests.pojo.web.Symptoms;
+import org.sormas.e2etests.services.SymptomService;
+import org.sormas.e2etests.state.ApiState;
+
+public class SymptomsTabSteps implements En {
+
+  private final WebDriverHelpers webDriverHelpers;
+  public static Symptoms symptoms;
+  public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("M/d/yyyy");
+
+  @Inject
+  public SymptomsTabSteps(
+      WebDriverHelpers webDriverHelpers,
+      SymptomService symptomService,
+      ApiState apiState,
+      @Named("ENVIRONMENT_URL") String environmentUrl) {
+    this.webDriverHelpers = webDriverHelpers;
+
+    When(
+        "I check the created data is correctly displayed on Symptoms tab page",
+        () -> {
+          Symptoms actualSymptoms = collectCasePersonData();
+          Truth.assertThat(actualSymptoms).isEqualTo(symptoms);
```

Comment:
should we create the SymptomsComparator?

Reply:
in this case no, because I have fully populated object, and it is very simple to be compared